tests/test_versioneer.py

```diff
@@ -60,18 +81,41 @@ def move_to_package_name(root):
     os.chdir(old)
 
 
+@pytest.fixture
+def move_to_another_package(root):
+    old = os.getcwd()
+    p = root / 'tests' / 'assets' / 'another_package'
+    os.chdir(p)
+    yield
+    os.chdir(old)
+
+
 def test_locate_package_and_readme(move_to_package_name):
-    v = Versioner()
+    v = VersionerSetup()
     assert v.PACKAGE == Path('src', 'package_name')
     assert v.path_to_changelog == Path('CHANGELOG.md')
 
 
+def test_locate_package_and_readme_non_setup(move_to_another_package):
+    v = VersionerNonSetup('app')
+    assert v.PACKAGE == Path('app')
+    assert v.path_to_changelog == Path('CHANGELOG.md')
+
+
 def test_current_version(move_to_package_name):
-    assert Versioner().current_version() == '0.1dev'
+    assert VersionerSetup().current_version() == '0.1dev'
+
+
+def test_current_version_non_setup(move_to_another_package):
+    assert VersionerNonSetup('app').current_version() == '0.1dev'
```

Comment:
there are a few tests that look the same except by the class they use. we can simplify them with this: https://docs.pytest.org/en/6.2.x/parametrize.html

since VersionerNonSetup takes an argument you can make it default to `app`

and then parametrize over the classes

Reply:
yes i tried that. For some I have added. But the ones in which `move_to_package_name' are included, the move_to doesn't seem to be working.
I had added a parameter `move_to` and values `move_to_another_package` and 'move_to_package_name`. But once these are parametrized the current directory stays stuck in the `pkgmt` folder instead of moving to respective packages